src/com/lilithsthrone/game/character/CharacterUtils.java

```diff
@@ -1255,6 +1255,12 @@ public static void randomiseBody(GameCharacter character, boolean randomiseAge)
 				if(Math.random()<=0.025f) {
 					character.addMilkModifier(FluidModifier.ADDICTIVE);
 				}
+				if(Math.random()<=0.025f) {
+					character.addMilkModifier(FluidModifier.FEMINIZING);
+				}
+				if(Math.random()<=0.025f) {
+					character.addMilkModifier(FluidModifier.MASCULINIZING);
+				}
```

Comment:
Should it be possible for an npc to generate with both modifiers? If so, perhaps both together would have a special effect, like a randomized change, or an androgynizing change (towards center), or an extremifying change (away from center).

Reply:
Hmmm.. valid point. Androgynizing for having both is probably the best solution. I was thinking about how it might be best to handle that.